vms/rpcchainvm/vm_client.go

```diff
@@ -41,6 +46,11 @@ var (
 	errUnsupportedFXs = errors.New("unsupported feature extensions")
 )
 
+const (
+	decidedCacheSize = 500
```

Comment:
No idea what the right number is here.

Reply:
If we see a lot of misses from the metrics, I think we can look into increasing the number then.